spring-kafka-avro/src/main/resources/application.properties

```diff
@@ -0,0 +1,4 @@
+spring.kafka.bootstrap-servers=localhost:9092
```

Comment:
See the other docker-compose based builds how to get the auto-generated port from docker-compose into your application.